pymc/tests/distributions/test_truncated.py

```diff
@@ -181,7 +181,11 @@ def test_truncation_discrete_random(op_type, lower, upper):
             assert np.any(xt_draws == upper)
     else:
         with pytest.raises(TruncationError, match="^Truncation did not converge"):
-            draw(xt)
+            # Rejections sampling with probability = (1 - p ** max_n_steps) ** sample_size =
+            # = (1 - 0.2 ** 3) ** 500 = 0.018
+            # Still, this probability can be too high to make this test pass with any seed.
+            random_seed = 2297228
+            draw(xt, random_seed=np.random.RandomState(random_seed))
```

Comment:
```suggestion
            draw(xt, random_seed=2297228)
```

Reply:
`draw` requires as input a RandomState, not an int 
https://github.com/pymc-devs/pymc/blob/main/pymc/sampling.py#L2274

Reply:
draw accepts int/RandomState or Generator.